### DISCUSSION TITLE
Property 'name' does not exist on type 'never[]'.

### DISCUSSION THREAD

**Discussion by Not-Sarthak** [refinedev/refine#4794] [Q&A]
Can anyone please help me solve the above mentioned error inside the my-profile page? Have been watching the tutorial by jsmastery but couldn't resolve it on the client side

**Comment by MahirMahdi** [comment]
Could you please share more details about the specific error or any relevant code snippets?

**Reply by Not-Sarthak** [reply]
![Screenshot 2023-08-07 172941](https://github.com/refinedev/refine/assets/92942966/f09cffdc-31aa-4e9e-bd8f-048716d76f21)
![Screenshot 2023-08-07 173107](https://github.com/refinedev/refine/assets/92942966/ae7d9c64-64cf-43a0-a781-d8402357090c)

![Screenshot 2023-08-07 173157](https://github.com/refinedev/refine/assets/92942966/de0521dd-7435-43d4-a38a-3cf77b58ef21)


Hey, do check the screenshots attached and let me know if you require something more :) thanks

**Reply by MahirMahdi** [reply]
From the snippets you've shared, it appears that there is a type mismatch between the "myProfile" and "Profile" props. Please ensure that the types of these props are the same.

**Comment by BatuhanW** [comment]
You need to pass type to `Array` generic. 

```
...
properties?: Array<string>
...
```
